ruby/BUILD.bazel

```diff
@@ -116,7 +103,7 @@ inline_sh_test(
     ],
     cmd = """
         pushd `dirname $(location Rakefile)`
-        RUBYLIB=../src:tests:. BAZEL=true rake test
```

Comment:
Moving this to the kokoro runner doesn't really have the same effect.  If someone were to try to run this manually via Bazel this flag would no longer be set and I think it would break.  The fact that you added it to the kokoro runner just kindof masks that breakage

Reply:
Should it go in `.blazerc` as well, then? While this target is unlikely to get much use outside of Kokoro (community Ruby developers are much more likely to just run `rake test` directly) or `travis-test.sh`, I agree we should strive to make it universally runnable.

Reply:
Yea that's a good idea

Reply:
Done